site/content/blog/2021-08-16-zap-fileupload-addon/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "ZAP FileUpload Add-on"
+description: "How to use ZAP FileUpload Add-on for finding Vulnerabilities in file upload functionality"
+images:
+- https://www.zaproxy.org/blog/2021-08-16-zap-fileupload-addon/images/fileupload-options-panel.png
+type: post
+tags:
+- blog
+date: "2021-08-16"
+addSocialPreview: true
+authors:
+    - preetkaran20
+---
+## Overview
+File upload is becoming a more and more essential part of any application, where the user is able to upload their photo, their CV, or a video showcasing a project they are working on. The application should be able to fend off bogus and malicious files in a way to keep the application and the users safe. Generally file upload functionality is quite complex to automate and has huge attack surface hence there is a need to automate the process and also secure it. So the FileUpload add-on has scan rule which is used to find vulnerabilities in File upload functionality and this blog explains on how to use it.
```

Comment:
The add-on was actually named `File upload Scan Rule`, either change the name here or in the add-on.

Reply:
Done